apps/frontend/src/libs/components/sidebar/sidebar-items.tsx

```diff
@@ -0,0 +1,29 @@
+import { AppRoute } from "~/libs/enums/app-route.enum.js";
+
+import { type sidebarItemType } from "./sidebar-item/type.js";
+
+const sidebarItems: sidebarItemType[] = [
+	{
+		icon: "project",
+		id: "1",
+		text: "Projects",
+		to: AppRoute.ROOT,
+	},
+	{
+		icon: "access",
+		id: "2",
+		text: "Access Management",
+	},
+	{
+		icon: "contributors",
+		id: "3",
+		text: "Contributors",
+	},
+	{
+		icon: "analytics",
+		id: "4",
+		text: "Analytics",
+	},
```

Comment:
Leave only project item
```suggestion
```

Reply:
But then other navigation items will not be displayed